### DISCUSSION TITLE
Connect Rclone

### DISCUSSION THREAD

**Discussion by Princess-Connect** [AlistGo/alist#4901] [General]
Hello, I have successfully linked my OneDrive edu account to rclone. However, I don't know how to assign it to alist's storage. Is there any way or documentation to help me with this? Thank you

**Comment by SeanHeuc** [comment]
if u want alist -> rclone -> onedrive,   check https://alist.nn.ci/guide/drivers/webdav.html, and use rclone serve webdav
if u want alist  -> onedrive, (rclone -> onedrive is not impactecd)   check https://alist.nn.ci/guide/drivers/onedrive.html
